src/models/document/document-content.test.ts

```diff
@@ -114,8 +113,11 @@ describe("DocumentContentModel", () => {
           { title: "Graph 1", content: { type: "Geometry", objects: [] } },
           { content: { type: "Text", format: "html", text: ["<p></p>"] } }
         ],
-        { title: "Table 1", content: { type: "Table",
-          columns: [{ name: "x", width: kDefaultColumnWidth }, { name: "y", width: kDefaultColumnWidth }] } },
+        { title: "Table 1",
+          content: {
+            type: "Table",
+          }
+        },
```

Comment:
```suggestion
        { title: "Table 1", content: { type: "Table" } },
```
Don't need the extra lines at this point.